src/Stratis.Bitcoin.Networks/StratisRegTest.cs

```diff
@@ -67,7 +68,9 @@ public StratisRegTest()
             var bip9Deployments = new StratisBIP9Deployments()
             {
                 // Always active on StratisRegTest.
-                [StratisBIP9Deployments.ColdStaking] = new BIP9DeploymentsParameters(1, BIP9DeploymentsParameters.AlwaysActive, 999999999)
+                [StratisBIP9Deployments.ColdStaking] = new BIP9DeploymentsParameters(1, BIP9DeploymentsParameters.AlwaysActive, 999999999),
```

Comment:
Hmm, you have the bit numbers incorrect for the regtest deployments I think

Reply:
Not that it matters much but yes thanks 😄